index.html

```diff
@@ -834,22 +834,29 @@ <h2>
         |screenOrientation|'s {{ScreenOrientation/[[angle]]}}, abort these
         steps.
         </li>
-        <li>Set |screenOrientation|'s {{ScreenOrientation/[[angle]]}} to
-        |angle|.
-        </li>
-        <li>Set |screenOrientation|'s {{ScreenOrientation/[[type]]}} to |type|.
-        </li>
-        <li>[=Fire an event=] named "<a data-link-for=
-        "ScreenOrientation">change</a>" at |screenOrientation|.
+        <li>[=Queue a task=] on the [=user interaction task source=] with
```

Comment:
Queue a task is the legacy algorithm. You want one of the ones that takes more arguments, such as queue a global task.